corehq/apps/linked_domain/tests/test_update_tableau_visualization.py

```diff
@@ -0,0 +1,45 @@
+from corehq.apps.reports.models import TableauVisualization, TableauServer
+from corehq.apps.linked_domain.local_accessors import get_tableau_visualizaton
+from corehq.apps.linked_domain.tests.test_linked_apps import BaseLinkedAppsTest
+from corehq.apps.linked_domain.updates import update_tableau_visualization
+
+
+class TestUpdateTableauVisualization(BaseLinkedAppsTest):
```

Comment:
maybe a good thing to add tests for more than one visualization? and then testing updating just one of them?